build.gradle

```diff
@@ -178,6 +178,7 @@ dependencies {
     testImplementation group: 'net.bytebuddy', name: 'byte-buddy', version: '1.14.2'
     testImplementation group: 'org.objenesis', name: 'objenesis', version: '3.2'
     testImplementation group: 'net.bytebuddy', name: 'byte-buddy-agent', version: '1.14.2'
+    api "org.opensearch:common-utils:${version}"
```

Comment:
Why we need this?

Reply:
we're importing some constants from common-utils for https client setup https://github.com/martin-gaievski/k-NN/blob/support-integ-tests-on-secured-cluster/src/testFixtures/java/org/opensearch/knn/ODFERestTestCase.java#L289-L300